src/modules/core/components/FileUpload/types.ts

```diff
@@ -11,9 +11,10 @@ export interface FileReaderFile {
 
 export interface UploadFile {
   file: File;
-  uploaded?: string;
+  uploaded?: string | boolean;
```

Comment:
why sometimes its a string and sometimes boolean? cant we make it more generic?

Reply:
Hmmm I added boolean because in other instances where the `uploaded` is used (AvatarUploadItem and UploadItem) that prop stores a string that represents the uploaded file, for whatever reason that I don't really know because from what I've seen this prop is used to check if the file was already uploaded. (Perhaps @rdig knows more about why this prop is used as a `string`)

Seeing that the prop is used on my case only the check if the .csv file was uploaded I decided it would be more appropriate for it to be a boolean, but I could change it to string and store whatever in it just for consistency sake.